### PR TITLE
Add Ready To Payout operations

### PR DESCRIPTION
**Pull request by ashkarpetin**
## Summary

In this PR was added definition of `/storefront/ready-to-payout` endpoint

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/RuleActionAdjustReadyToPayout.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+allOf:
+  - $ref: ./RuleAction.yaml
+  - type: object
+    description: Adjust ready to payout.
+    properties:
+      prioritizeActivePaymentInstruments:
+        type: boolean
+        default: false
+        description: If set as `true`, the payment methods associated to the customer's active payment instruments are displayed at the top of the list.
```
Comment (noon-dawg) [inline openapi/components/schemas/RuleActionAdjustReadyToPayout.yaml:9]: nitpick: [boolean fields](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#boolean-fields)
```suggestion
        description: Specifies if payment methods, that are associated with the customer's active payment instruments, are displayed at the top of the list.
```

File: openapi/components/schemas/RuleActionAdjustReadyToPayout.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      paymentMethods:
+        type: array
+        description: Ordered list of payment methods allowed. If empty, then no payment methods are allowed.
```
Comment (noon-dawg) [inline openapi/components/schemas/RuleActionAdjustReadyToPayout.yaml:12]: suggestion: for readability
```suggestion
        description: Ordered list of allowed payment methods. If this field is empty, no payment methods are allowed.
```

File: openapi/paths/storefront/ready-to-payout.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    - PublishableApiKey: []
+  description: |-
+    Retrieves available payment methods for a specific payout amount and currency.
+
+    The order in which the payment methods are displayed to the customer must be the same as the order in the response.
+
+    The list of payment methods is generated from available gateway accounts and the last matched `adjust-ready-to-payout` action on the `ready-to-payout-requested` event.
+    If no rules match for the specific request, all methods supported by the gateway accounts are sent.
+
+    To invert this behavior, place an all matching rule at the end of the `ready-to-payout-requested` event in the rules engine,
```
Comment (noon-dawg) [inline openapi/paths/storefront/ready-to-payout.yaml:19]: nitpick typo
```suggestion
    To invert this behavior, place an all matching rules at the end of the `ready-to-payout-requested` event in the rules engine,
```

**Review by noon-dawg — APPROVED** [review]
Looks good on the docs side

**Review by mattsanf — APPROVED** [review]

**Review by giokaxo — APPROVED** [review]

**Pull request merged**